bin/celeborn-ratis

```diff
@@ -71,6 +71,8 @@ function main {
   CELEBORN_RATIS_SHELL_JAVA_OPTS+=" -Dorg.apache.jasper.compiler.disablejsr199=true"
   CELEBORN_RATIS_SHELL_JAVA_OPTS+=" -Djava.net.preferIPv4Stack=true"
   CELEBORN_RATIS_SHELL_JAVA_OPTS+=" -Dorg.apache.ratis.thirdparty.io.netty.allocator.useCacheForAllThreads=false"
+  # Netty is supported by default
+  CELEBORN_RATIS_SHELL_JAVA_OPTS+=" -Draft.rpc.type=NETTY"
```

Comment:
some background:

- `netty` is chosen by coincidence, based on our research, `grpc` is selected by most of the projects that use ratis, maybe we need to investigate if we can switch to `grpc` at some time
- the ratis shell support changing rpc type by args

Reply:
ok, get it.

Reply:
> some background:
> 
> * `netty` is chosen by coincidence, based on our research, `grpc` is selected by most of the projects that use ratis, maybe we need to investigate if we can switch to `grpc` at some time
> * the ratis shell support changing rpc type by args

According to the latest test, using `grpc` is runnable but with higher election time than netty. There are no clear benefits to using `grpc`.